src/twisted/test/test_sslverify.py

```diff
@@ -267,20 +268,28 @@ def dataReceived(self, data):
         def connectionLost(self, reason):
             self.lostReason = reason
 
+    clientWrappedProto = ListeningClient()
+    serverWrappedProto = GreetingServer()
+
+    cF = protocol.Factory()
```

Comment:
I would prefer not to use abbreviations and call this clientFactory. ... why not clientFact ? :)

...same for clientWrappedProto ... why not cWP ?

------

I see that the existing code uses abbreviations, but I don't think that this is a good reason to continue to write new code in this way :)